docs/tls/README.md

```diff
@@ -8,7 +8,7 @@ In this document, we will configure Minio servers with TLS certificates for both
 
 ## 2. Configure with existing certificates
 
-Assuming that you are already having private and public certificates, you will need to copy them under `certs` in your Minio config directory using the names `private.key` and `public.crt` for key and public certificates respectively.
+Assuming that you are already having private and public certificates, or passwordless PKCS#12 certificate, you will need to copy them under `certs` in your Minio config directory using the names `private.key` and `public.crt` for key and public certificates respectively, or `cert.pfx` in case of PKCS#12 certificate.
```

Comment:
> Is there a password based approach as well?

Yes.

> should we mention that we don't support it?

I mentioned as `passwordless`. Do you want more elaborate message?

> Have you explored about can we support it? @balamurugana

As a password, for security reasons, it should be read from stdin   Other ways are insecure.  Its UX decision how password needs to be read.